### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Arcoders**

### CODE REVIEW

File: staff/ismael-haytam/cv-styled/index.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                <h1>Resumen</h1>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Atque facilis, mollitia provident, quisquam nostrum maxime officiis excepturi fugiat soluta illo eligendi magnam maiores. Eos quos animi magni necessitatibus dolores totam.</p>
+            </article>
+        </section>
+    
+        <section class="container cornsilk">
+    
+            <h2 align="center">Habilidades</h2>
+        
+            <div class="info">
```
Comment (anasanjuan) [inline staff/ismael-haytam/cv-styled/index.html:29]: I see too many divs, but not sure if you could do it more semantic

File: staff/ismael-haytam/cv-styled/index.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>CV</title>
+</head>
+<body>
+    
+    <div class="app">
```
Comment (anasanjuan) [inline staff/ismael-haytam/cv-styled/index.html:12]: great code! maybe you can try to apply BEM if you have spared time :)

File: staff/ismael-haytam/cv-styled/css/style.css [hunk 3]
```diff
@@ -0,0 +1,226 @@
+@import url('https://fonts.googleapis.com/css?family=Bai+Jamjuree');
```
Comment (maryammdot) [inline staff/ismael-haytam/cv-styled/css/style.css:1]: very nice css!!

File: staff/ismael-haytam/cv-styled/index.html (hunk 1 above)
Comment (Arcoders) [reply staff/ismael-haytam/cv-styled/index.html:29]: Done ^_^

File: staff/ismael-haytam/cv-styled/css/style.css (hunk 3 above)
Comment (Arcoders) [reply staff/ismael-haytam/cv-styled/css/style.css:1]: thanks ^_^

**Pull request merged**